types/KBaseStructure.ExperimentalProteinStructure/spec.json

```diff
@@ -1,10 +1,12 @@
 {
     "ver":"0.1.0",
     "view_method_ids":[
+        "NarrativeViewers/view_exp_proteinstructure"
```

Comment:
Note that this function doesn't exist yet.

Reply:
For this one and the next, I may remove them for now.  It's a developing/evolving process, Chris leans toward merging these two into one.